dhctl/pkg/system/ssh/session/session.go

```diff
@@ -106,6 +109,58 @@ func (s *Session) ChoiceNewHost() {
 	s.selectNewHost("")
 }
 
+func (s *Session) AddAvailableHosts(hosts ...string) {
+	defer s.lock.Unlock()
+	s.lock.Lock()
+
+	availableHostsMap := make(map[string]struct{}, len(s.availableHosts))
+
+	for _, host := range s.availableHosts {
+		availableHostsMap[host] = struct{}{}
+	}
+
+	for _, host := range hosts {
+		availableHostsMap[host] = struct{}{}
+	}
+
+	availableHosts := make([]string, 0, len(availableHostsMap))
+
+	for host := range availableHostsMap {
+		availableHosts = append(availableHosts, host)
+	}
+
+	s.availableHosts = availableHosts
+
+	s.resetUsedHosts()
+	s.selectNewHost("")
+}
```

Comment:
please add unit tests for this functions

Reply:
Added